src/intl.ts

```diff
@@ -112,6 +112,18 @@ export const commonMessages = defineMessages({
     defaultMessage: "Upload image",
     description: "button"
   },
+  userStatus: {
```

Comment:
These messages should be in local messages file, not it the main / common messages.